lib/command/init.rb

```diff
@@ -79,7 +80,11 @@ def init_aozoraepub3(force = false)
         puts "<bold><red>#{"!!!WARNING!!!".center(70)}</red></bold>".termcolor
         puts "AozoraEpub3の構成ファイルを書き換えます。narouコマンド用に別途新規インストールしておくことをオススメします"
       end
-      aozora_path = @aozora_dirname or ask_aozoraepub3_path
+      if @aozora_dirname
+        path = check_aozoraepub3_path(@aozora_dirname)
+        print "\n<bold><green>指定されたフォルダにAozoraEpub3がありません。</green></bold>\n".termcolor if !path
```

Comment:
__[rubocop]__
* Style/NegatedIf: Favor `unless` over `if` for negative conditions. [:ref](https://github.com/bbatsov/ruby-style-guide#unless-for-negatives)

<!-- Comment by [SideCI](https://sideci.com) -->